doc/source/categorical.rst

```diff
@@ -89,6 +104,56 @@ By passing a :class:`pandas.Categorical` object to a `Series` or assigning it to
     df["B"] = raw_cat
     df
 
+Categorical data has a specific ``category`` :ref:`dtype <basics.dtypes>`:
+
+.. ipython:: python
+
+    df.dtypes
+
+DataFrame Creation
+~~~~~~~~~~~~~~~~~~
+
+Columns in a ``DataFrame`` can be batch converted to categorical, either at the time of construction
```

Comment:
Columns -> All columns

And maybe can start this section with saying that single column can be converted similar to Series (df['col'] = df['col'].astype(category)), to then go to what to do if you want multiple / all columns to be categorical

Reply:
- Remove the comma after "categorical"

- "at the time of" -> "during"

Reply:
The previous "Series Creation" section actually does single column conversion, so I just mentioned that section instead of repeating the code/examples.